src/murmur/ServerDB.cpp

```diff
@@ -80,6 +80,9 @@ void ServerDB::loadOrSetupMetaPKBDF2IterationsCount(QSqlQuery &query) {
 }
 
 ServerDB::ServerDB() {
+	if (Meta::mp.qsDBDriver != QLatin1String("QMYSQL") && Meta::mp.qsDBDriver != QLatin1String("QSQLITE")  && Meta::mp.qsDBDriver != QLatin1String("QPSQL")) {
+		qFatal("ServerDB: invalid DB driver specified: '%s'. Murmur only supports QSQLITE, QMYSQL, and QPSQL.");
```

Comment:
You're using a %s format string here, but you don't provide any input.

You should use `qPrintable(Meta::mp.qsDBDriver)`